src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -453,12 +451,12 @@ static bool IsApplicableBase(const ConvolutionContext& params)
     if(!params.rmv.IsV3())
         return false;
 
+    const auto target = params.GetStream().GetTargetProperties();
+    if(target.Xnack() && *target.Xnack())
+        return false;
+
     const auto name = params.GetStream().GetDeviceName();
-#if WORKAROUND_ISSUE_1093
-    if(!(StartsWith(name, "gfx9") || StartsWith(name, "gfx10")) || name == "gfx90a")
-#else
     if(!(StartsWith(name, "gfx9") || StartsWith(name, "gfx10")))
-#endif
         return false;
     if(params.IsFp16() &&
        !(StartsWith(name, "gfx906") || StartsWith(name, "gfx908") || StartsWith(name, "gfx1011") ||
```

Comment:
@Slimakanzer Would this work on gfx90a?

Reply:
@atamazov No, currently _ConvBinWinogradRxSf2x3_ does not support fp16 on gfx90a due to verification errors. I'm working on it.